Lesson 1: Task1: Simple guess number game

VladFear opened the pull request:
This PR is needed for adding simple guess game, which is very interesting and elaborating.

galkinletter commented on 01-Git_Bash_Make/guess_a_number.sh:
```diff
@@ -0,0 +1,20 @@
+  echo "You Win! Congratulations."
+else
+  echo "You lose... Try again."
+fi
```
add empty line at the end of file
```suggestion
fi

```

galkinletter reviewed (changes requested):
Fix empty line

VladFear committed:
Lesson 1: Task 1: Create main function

Signed-off-by: VladFear <onlylastfear1@gmail.com>

VladFear committed:
Lesson 1: Task 1: Add opportunity to enter value

Signed-off-by: VladFear <onlylastfear1@gmail.com>

VladFear committed:
Lesson 1: Task 1: Add check for input value

Signed-off-by: VladFear <onlylastfear1@gmail.com>

VladFear committed:
Lesson 1: Task 1: Add randomly generated value and it's output

Signed-off-by: VladFear <onlylastfear1@gmail.com>

VladFear committed:
Lesson 1: Task 1: Add comparison between values

Add comparison for randomly generated value and user input.
The result is message about winning or loosing game.

Signed-off-by: VladFear <onlylastfear1@gmail.com>

VladFear commented:
@galkinletter 
The code was rewritten from Bash to C++. Check it please.

StanislavGoncharovGL commented on 01-Git_Bash_Make/main.cpp:
```diff
@@ -0,0 +1,10 @@
+
+int main(int argc, char** argv)
+{
+	cout << "Enter value: ";
```
I think It is better to move the line output `cout << "Enter value: "` to the next commit, because functionality it is close to the 2nd commit.

VladFear committed:
Lesson 1: Task 1: Add prompt for user input

Signed-off-by: VladFear <onlylastfear1@gmail.com>

VladFear commented:
@StanislavGoncharovGL 
Changes have been done. Please have a look.

StanislavGoncharovGL commented:
> @StanislavGoncharovGL
> Changes have been done. Please have a look.

Looks Ok, tnx

VladFear commented:
@StanislavGoncharovGL 
Can PR be merged?
